### ISSUE TITLE
Playlist: can not set the duration to under 1 second

### ISSUE THREAD

**Issue by KlausHans** [Aircoookie/WLED#3359]
### What happened?

Based on https://github.com/Aircoookie/WLED/issues/346#issuecomment-1219882212 i tried to set a duration for a playlist entry to 0.1 s. But the minimum duration i can set in the gui is 1 s. If i change to json view and set "dur":[1] (0.1 s) the value doesn't get saved. Instead the default value of 100 (10 s) is saved.
Tested Firefox and Chromium.

### To Reproduce Bug

- create preset
- add the preset in a playlist
- try to set the duration of the added preset to <1 s (e.g. 0.1 s)

Some screenshots of my setup:
Preset:
![grafik](https://github.com/Aircoookie/WLED/assets/9212982/4ef5042e-f7f8-4a1d-add7-74858ef6afbd)

Playlist with fresh input of "dur":[1]:
![grafik](https://github.com/Aircoookie/WLED/assets/9212982/936c29f2-4e32-49e4-9bd4-c5f4b9b4cd21)
after clicking save and open the playlist again;
![grafik](https://github.com/Aircoookie/WLED/assets/9212982/4fd7f028-6839-480e-9025-d5f0bf512bcf)

### Expected Behavior

I can set a duration lower than 1 second.

### Install Method

Binary from WLED.me

### What version of WLED?

0.14.0-b4

### Which microcontroller/board are you seeing the problem on?

ESP32

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by blazoncek** [comment]
Minimum duration value is 0.2s. This is a design choice.
Tested on multiple units.

**Comment by KlausHans** [comment]
0.2 works indeed. Thank you for clarifying. This info could be added to the documentation.

**Comment by KlausHans** [comment]
It was both. A min constraint on the input in the Ui is a good idea.

**Comment by blazoncek** [comment]
> @blazoncek not sure if this was about a value entered in a field in web UI, or just direct json in presets? If it's about web UI, maybe we could add a "min" to the input field?

UI already has the "min" property and a "step" property (min="0.2" step="0.1"). This is converted to 1/10ths of a second and submitted in JSON.

**Comment by KlausHans** [comment]
Ok but it doesn't show any hint or error if i put 0.1 in.

**Comment by blazoncek** [comment]
> Ok but it doesn't show any hint or error if i put 0.1 in.

True, I noticed that. There is a validation, but does not work as expected.
